tools/query_verification/src/test/java/com/google/bigquery/QueryVerifierTest.java

```diff
@@ -102,4 +103,34 @@ public void testGetMultipleTableIdFromDdlSchema() {
         assertEquals(tableId.getTable(), "secondTable");
     }
 
+    @Test
+    public void testGetJobInfoFromQuery() {
+        String queryContents = "SELECT * FROM table";
+        QueryVerificationQuery query = QueryVerificationQuery.create(queryContents, "");
+        List<JobInfo> jobInfos = QueryVerifier.getJobInfosFromQuery(query, true);
+
+        assertEquals(jobInfos.size(), 1);
+
+        QueryJobConfiguration queryJobConfiguration = jobInfos.get(0).getConfiguration();
+        assertEquals(queryJobConfiguration.getQuery(), "SELECT * FROM table");
+    }
+
+    @Test
+    public void testGetMultipleJobInfosFromQuery() {
+        String queryContents = "SELECT * FROM table1;\nSELECT column1 FROM table2; SELECT column2 FROM table2;";
+        QueryVerificationQuery query = QueryVerificationQuery.create(queryContents, "");
+        List<JobInfo> jobInfos = QueryVerifier.getJobInfosFromQuery(query, true);
+
+        assertEquals(jobInfos.size(), 3);
+
+        List<String> queries = jobInfos.stream().map(jobInfo -> {
+            QueryJobConfiguration queryJobConfiguration = jobInfo.getConfiguration();
```

Comment:
I would inline this into the return and then remove the {} from the lambda so it looks like ```jobInfo -> jobInfo.getConfiguration().getQuery()``` (I think this is valid syntax)

Reply:
Only reason I didn't do that is because that actually requires manual casting since `.getConfiguration()` returns a `JobConfiguration` type (super class of `QueryJobConfiguration`) and doesn't have a `.getQuery()` method

Reply:
```QueryJobConfiguration queryJobConfiguration = jobInfo.getConfiguration();``` doesn't seem to have a downcast, is it not needed when you assign like this?

Reply:
So it looks like `JobConfiguration` is an abstract class and `.getConfiguration()` instead returns `<C extends JobConfiguration>`